frontend/src/components/IntroBanner.vue

```diff
@@ -43,6 +31,19 @@ export default {
     };
     return {
       dismiss,
+      content: formatStringTemplate(
+        t("intro.content"),
+        `<span class="text-accent">${t("intro.quickstart")}</span>`,
+        `<a class="normal-link" href="https://docs.bytebase.com" target="__blank">
+          ${t("intro.doc")}
+        </a>`,
+        `<a
+          class="normal-link"
+          href="https://github.com/bytebase/bytebase/issues"
+          target="__blank"
+          >${t("intro.issue")}</a
+        >`
+      )
```

Comment:
Would you try [Vue I18n - Component Interpolation](https://vue-i18n.intlify.dev/guide/advanced/component.html#basic-usage)?

Reply:
Not yet. will take a try

Reply:
It works. THX